lbz/lambdas/broker.py

```diff
@@ -0,0 +1,42 @@
+from typing import Callable, Dict, Union
+
+from lbz.handlers import BaseHandler
+from lbz.lambdas.enums import LambdaResult
+from lbz.lambdas.response import LambdaResponse, lambda_error_response
+from lbz.misc import get_logger
+
+logger = get_logger(__name__)
+
+
+class LambdaBroker(BaseHandler):
+    def __init__(
+        self,
+        mapper: Dict[str, Callable[..., LambdaResponse]],
+        event: dict,
+        context: object = None,
+    ) -> None:
+        self.event = event
+        self.context = context
+        self.mapper = mapper
+        self.data = event.get("data")
+
+    def handle(self) -> LambdaResponse:
+        handler = self._get_handler()
+        if isinstance(handler, dict):
+            return handler
+        if self.data:
```

Comment:
[#966041983](https://github.com/pdyba/lambdalizator/pull/64#discussion_r966041983):
> This is not correct as it can be e.g. an empty dict - as for me, you should set the whole `event` at the object level and try to fetch data here directly.